docs/install.rst

```diff
@@ -1,7 +1,7 @@
 .. _plasmapy-install:
 
 *******************
-Installing PlasmaPy
+Installing PlasmaPy ☀️
 *******************
```

Comment:
```suggestion
Installing PlasmaPy ☀️
**********************
```
📝 ...since title underlines and overlines have to be the same length as the titles, and emojis seem to require two spaces.